Upstream: yes

From f4ec3ee6da0efeea7a62a6ea3a892ac4bb374182 Mon Sep 17 00:00:00 2001
From: David Faure <faure@kde.org>
Date: Tue, 3 Apr 2018 00:21:46 +0200
Subject: [PATCH] Fix compilation with Qt 5.11 (missing include)

---
 src/svnfrontend/blamedisplay.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/svnfrontend/blamedisplay.cpp b/src/svnfrontend/blamedisplay.cpp
index 6fbc7d0e..345701de 100644
--- a/src/svnfrontend/blamedisplay.cpp
+++ b/src/svnfrontend/blamedisplay.cpp
@@ -30,6 +30,7 @@
 #include <KTextEdit>
 #include <KTreeWidgetSearchLine>
 
+#include <QAction>
 #include <QBrush>
 #include <QFontDatabase>
 #include <QInputDialog>
-- 
2.21.0

